aws_lambda_powertools/utilities/idempotency/config.py

```diff
@@ -41,3 +46,7 @@ def __init__(
         self.use_local_cache = use_local_cache
         self.local_cache_max_items = local_cache_max_items
         self.hash_function = hash_function
+        self.lambda_context: Optional[LambdaContext] = lambda_context
+
+    def register_lambda_context(self, lambda_context: LambdaContext):
+        self.lambda_context = lambda_context
```

Comment:
```suggestion
    def register_lambda_context(self, lambda_context: LambdaContext):
        """Include Lambda context to calculate remaining time before it times out"""
        self.lambda_context = lambda_context
```

Docstring for public method. There's gotta be a better wording but can't think of one right now